new-log-viewer/src/components/StatusBar/LogLevelSelect/index.tsx

```diff
@@ -0,0 +1,273 @@
+ * @return
+ */
+const ClearFiltersOption = ({onClick}: ClearFiltersOptionProps) => (
+    <Option
+        value={-1}
+        onClick={onClick}
+    >
+        <ListItemDecorator>
+            <CloseIcon/>
+        </ListItemDecorator>
+        Clear filters
+    </Option>
+);
+
+/**
+ * Renders a dropdown box for selecting log levels.
+ *
+ * @return
+ */
+const LogLevelSelect = () => {
+    const [selectedLogLevels, setSelectedLogLevels] = useState<LOG_LEVEL[]>([]);
+
+    const handleRenderValue = (selected: SelectValue<SelectOption<LOG_LEVEL>, true>) => (
+        <Box className={"log-level-select-render-value-box"}>
+            <Chip className={"log-level-select-render-value-box-label"}>
+                Log Level
+            </Chip>
+            {selected.map((selectedOption) => (
+                <LogLevelChip
+                    key={selectedOption.value}
+                    name={selectedOption.label as string}
+                    value={selectedOption.value}/>
+            ))}
+        </Box>
+    );
+
+    const handleCheckboxClick = useCallback((ev: React.MouseEvent<HTMLInputElement>) => {
+        ev.preventDefault();
+
+        const target = ev.target as HTMLInputElement;
+        const value = Number(target.value) as LOG_LEVEL;
+        let newSelectedLogLevels: LOG_LEVEL[];
+        if (selectedLogLevels.includes(value)) {
+            newSelectedLogLevels = selectedLogLevels.filter((logLevel) => logLevel !== value);
+        } else {
+            newSelectedLogLevels = [
+                ...selectedLogLevels,
+                value,
+            ];
+        }
+        setSelectedLogLevels(newSelectedLogLevels.sort((a, b) => a - b));
+    }, [selectedLogLevels]);
+
+    const handleSelectChange = useCallback((
+        ev: React.MouseEvent | React.KeyboardEvent | React.FocusEvent | null
+    ) => {
+        if (null === ev) {
+            setSelectedLogLevels([]);
+
+            return;
+        }
+
+        const target = ev.target as HTMLElement;
+        const selectedValue = Number(target.dataset.value);
+        setSelectedLogLevels(range(selectedValue, 1 + MAX_LOG_LEVEL));
+    }, []);
+
+    const handleSelectClearButtonClick = () => {
+        handleSelectChange(null);
+    };
+
+    const handleSelectClearButtonMouseDown = (ev: React.MouseEvent<HTMLButtonElement>) => {
+        ev.stopPropagation();
+    };
+
+    return (
+        <Select
+            className={"log-level-select"}
+            multiple={true}
+            renderValue={handleRenderValue}
+            size={"sm"}
+            value={selectedLogLevels}
+            variant={"soft"}
+            indicator={0 === selectedLogLevels.length ?
+                <KeyboardArrowUpIcon/> :
+                <Tooltip title={"Clear filters"}>
+                    <IconButton
+                        variant={"plain"}
+                        onClick={handleSelectClearButtonClick}
+                        onMouseDown={handleSelectClearButtonMouseDown}
```

Comment:
What's this for?

Reply:
To my recollection it attempted to avoid toggling the listbox when the button is clicked although I can no longer reproduce the behaviour with this handler removed now. 
Let me remove it altogether to avoid confusions.